starknet_py/net/full_node_client.py

```diff
@@ -244,7 +244,9 @@ async def deploy(self, transaction: Deploy) -> DeployTransactionResponse:
         res = await self._client.call(
             method_name="addDeployTransaction",
             params={
-                "contract_address_salt": transaction.contract_address_salt,
+                "contract_address_salt": convert_to_felt(
```

Comment:
Why is it convert to felt here and hex below?

Reply:
Because it was updated in the last devnet release and it is compatible with RPC specification